pandas/core/window.py

```diff
@@ -659,6 +660,26 @@ def validate(self):
         else:
             raise ValueError("Invalid window {0}".format(window))
 
+    def _validate_aggregate(self, arg, *args, **kwargs):
+        # supported weighted window functions
```

Comment:
rather that this (listing methods)

you can look a bit more into where the methods are called and catch an invokation error (AttributError) for example

we already have quite a bit of code to facilitate this and don’t want to add more layers